content-src/components/Spotlight/Spotlight.scss

```diff
@@ -55,10 +71,16 @@
     top: $spotlight-icon-gutter;
   }
 
+  .spotlight-feedname {
+    color: $mid-light-grey;
+    line-height: 1.8;
+    font-size: 10px;
+  }
+
   .spotlight-title {
     margin: 0 0 $spotlight-title-margin-bottom;
     font-size: inherit;
-    line-height: inherit;
+    line-height: 1.3;
```

Comment:
I'm going to spin this off to another bug, probably on Monday, for two reasons:

* this problem already exists on master
* this particular solution looks nicer, but has the disadvantage of cropping the more important info (the title) rather than the less important info (the snippet), so I suspect we're going to want some design review...